packages/manager/modules/anthos/src/dashboard/routing.js

```diff
@@ -110,7 +110,11 @@ export default /* @ngInject */ ($stateProvider) => {
         });
 
         if (message) {
-          promise.then(() => displayAlerterMessage(type, message));
+          promise.then(() => {
+            displayAlerterMessage(type, message);
+            // eslint-disable-next-line no-param-reassign, no-multi-assign
+            $document[0].body.scrollTop = $document[0].documentElement.scrollTop = 0;
```

Comment:
Nitpick: may use 
scrollToTop() {
    this.$location.hash(this.PAGE_SECTIONS.BACK_BUTTON);
    this.$anchorScroll();
  }
as mentioned in /Users/raadired/ovh-/manager/apps/web/client/app/hosting/database/order/private/hosting-database-order-private.controller.js and in other regions

Reply:
how about this ?
```suggestion
            $location.hash(ANTHOS_TENANT_ALERTER);
            $anchorScroll();
```

Reply:
Fixed :+1: